### ISSUE TITLE
we will be creating a website using html css js to host our notes and resources for students

### ISSUE THREAD

**Issue by fury-coder** [fury-coder/Coding-Library#10]

**Comment by fury-coder** [comment]
firstly add index.html

**Comment by ujju20** [comment]
Can I work on this issue?

**Comment by fury-coder** [comment]
yes u can some points
under html create -
1. Coding Library heading 
2. nice background picture
3. decent ui
4. and sub headings like Java C++ python under which we will later push notes..
5. // Happy Hacking

**Comment by pcpriyankapc** [comment]
Please assign me this I can create a website for the notes and resources.

**Comment by ujju20** [comment]
> Please assign me this I can create a website for the notes and resources.

Bro, we can divide the work
Can we @fury-coder ??

**Comment by fury-coder** [comment]
yes go ahead.. we promote collaboration ..
u can divide the work and contribute..

**Comment by pcpriyankapc** [comment]
> > Please assign me this I can create a website for the notes and resources.
> 
> Bro, we can divide the work Can we @fury-coder ??

Sure!

**Comment by fury-coder** [comment]
yes @pcpriyankapc u definitely can..that's what open source is ..collaborating

**Comment by ujju20** [comment]
> > > Please assign me this I can create a website for the notes and resources.
> > 
> > 
> > Bro, we can divide the work Can we @fury-coder ??
> 
> Sure!

So Which Part of issue you are taking?

**Comment by ujju20** [comment]
![Screenshot from 2021-10-06 15-33-13](https://user-images.githubusercontent.com/64632969/136182557-41291b5d-549a-4857-b4fb-a24e3f4761bd.png)
This is the Basic UI of the web Page I have made 
How is it?

**Comment by fury-coder** [comment]
nice @ujju20 .. try to give a nice background picture

**Comment by ujju20** [comment]
How is it?
![Screenshot from 2021-10-07 15-19-06](https://user-images.githubusercontent.com/64632969/136361554-1c946248-a3b4-4b6c-befb-6693b2fe7f09.png)

**Comment by fury-coder** [comment]
ok nice but look for a better one..otherwise raise pr with it only. no problem
